docs/source/getting_started_guide/installing_receptor.rst

```diff
@@ -0,0 +1,15 @@
+###################
+Installing Receptor
+###################
+
+1. `Download receptor <https://github.com/ansible/receptor/releases>`_
+2. Install receptor
+
+.. code-block:: bash
+
+    pip install receptorctl
```

Comment:
receptor also needs to be installed on the system in order for receptorctl to work

Reply:
Added text to clarify the separate steps for receptor and receptorctl